etsin_finder/frontend/js/components/qvain/files/directoryForm.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { inject, observer } from 'mobx-react'
+import styled from 'styled-components';
+import Translate from 'react-translate-component'
+import {
+  SaveButton,
+  CancelButton
+} from '../general/buttons'
+import { Label, CustomSelect } from '../general/form'
+import { Container } from '../general/card'
+import { getLocalizedOptions } from '../utils/getReferenceData';
+
+export class DirectoryFormBase extends Component {
```

Comment:
The component for the file metadata editing, except for directories, as they are a bit different from files.